modules/DMZ/data_kit/filler_strategy.py

```diff
@@ -0,0 +1,45 @@
+import pandas as pd
+import dataset_insight
+import data_filler
+
+class FillerStrategy(object):
+
+    def __init__(self):
+        self.file_name = None
+        self.avg_thresh = None
+        self.value_thresh = None
+        self.fill_value = None
+        self.pandas_dataset = None
+        self.missing_ratios_dict = None
+
+        self.get_config_options()
+        self.get_dataset()
+        self.get_missing_ratios_dict()
+        self.run_fillers()
+
+    def get_config_options(self):
+        self.file_name = "titanic_train.csv"
+        self.avg_thresh = .50
+        self.value_thresh = .70
+        self.fill_value = -9999
+
+    def get_dataset(self):
+        self.pandas_dataset = pd.read_csv(self.file_name, sep=None)
```

Comment:
If sep is left to none, I believe it autodetects the separator, @ASAAR can correct me if I'm wrong.